build.gradle

```diff
@@ -49,6 +49,12 @@ allprojects {
         }
     }
 
+    checkstyle {
+        configFile = file("${project.rootDir}/google_checks.xml")
+        reportsDir = file("${project.rootDir}/build/reports/checkstyle/")
+        toolVersion = '6.14'
+    }
+
     check.dependsOn jacocoTestReport
```

Comment:
Maybe `check.dependsOn build`? This should work unless there is a circular task dependency